core-services/prow/02_config/stolostron/governance-policy-addon-controller/_prowconfig.yaml

```
@@ -11,8 +11,10 @@ branch-protection:
                 dismiss_stale_reviews: true
               required_status_checks:
                 contexts:
-                - KinD tests (1.18, v1.19.16)
-                - KinD tests (1.18, latest)
+                - KinD tests / KinD tests (latest, true)
+                - KinD tests / KinD tests (latest, false)
+                - KinD tests / KinD tests (v1.19.16, true)
+                - KinD tests / KinD tests (v1.19.16, false)
```

Same comment as above. Won't the "Linting and Unit Tests" be coming midstream also?
```suggestion
                - KinD tests (latest, true)
                - KinD tests (latest, false)
                - KinD tests (v1.19.16, true)
                - KinD tests (v1.19.16, false)
                - Linting and Unit tests
```